OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -182,7 +182,7 @@ public CPos GetRandomBaseCenter()
 		readonly IPathFinder pathfinder;
 
 		readonly Func<Actor, bool> isEnemyUnit;
-		readonly Predicate<Actor> unitIsDeadOrHasNewOwner;
+		readonly Predicate<Actor> unitInWorldIsDeadOrHasNewOwner;
```

Comment:
Should be something like `unitOutOfWorldIsDeadOrHasNewOwner` since it checks `!a.IsInWorld`.